Auto start timer and auto change between times

ataideal opened the pull request:
Seted focus time to 25 min, break to 5 min and long break to 30 min
Changing background colors and titles
Reset time now is reseting current break/focus time
Counting break/focus times

ataideal committed:
Auto start timer and auto change between times
Seted focus time to 25 min, break to 5 min and long break to 30 min
Changing background colors and titles
Reset time now is reseting current break/focus time
Counting break/focus times

siderlan reviewed (changes requested):
Great job! Thanks for your contribution.

I revised your code and I propose you some changes to approve it.

- Set the background color of the buttons to transparent and set the color to #fff.
- Is it possible that the forward method changes the current state? Because if you pause the time and click on the forward button the time will be completed but the state will just change after start the timer again.
- Add spec tests if you can

ataideal commented:
Thank's for the feedback @siderlan 
I really forgot about the forward time when it's paused.
I going to work on these changes.

ataideal committed:
Seted colors as was ordered
Fixed foward function to skip current time instantaneously
Added tests to verify inital state and foward function

siderlan reviewed (approved):
That's Ok
